_posts/2016-06-09-bayesian-quadrature.markdown

```diff
@@ -5,10 +5,10 @@ categories: jekyll update
 img: bayesian_quadrature.png
 ---
 
-Bayesian quadrature  is an active learning method for the value of an integral given queries of the integrand 
+Bayesian quadrature [[1, 2]](#refereces-on-quadrature) is an active learning method for the value of an integral given queries of the integrand 
```

Comment:
The first sentence is the one that appears in the card in the landing page and it doesn't make much sense having the references there (that's why in all the blogs we start adding the references in the second paragraph.)

Reply:
Changed.